sktime/forecasting/nnetsaucedeepmts.py

```diff
@@ -0,0 +1,390 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Interface to estimators from nnetsauce by Techtonique."""
+
+import numpy as np
+import pandas as pd
+from nnetsauce import DeepMTS as DeepMTS0
```

Comment:
After you move the specific dependency to extras, this line will fail, as not olall our users and our CI are going to install nnetsauce. So please remove this import statement here, and instead add it inside one of the methods.

To ensure that works reliably, use the tag `python_dependencies` (and `python_version`). It'll ensure that It'll proceed from `__init__` only if requirements are satisfied.